tests_integration/nativeapp/test_init_run.py

```diff
@@ -38,7 +46,7 @@ def test_nativeapp_init_run_without_modifications(
 ):
     project_name = "myapp"
     with nativeapp_project_directory(test_project):
-        result = runner.invoke_with_connection_json(["app", "run"])
+        result = runner.invoke_with_connection_json(command.split())
```

Comment:
you imported `shlex.split` but it's not used here?

Reply:
Fixed